.github/codecov.yml

```diff
@@ -11,3 +11,13 @@ flag_management:
     - name: jest-storefront
       paths:
         - src/Storefront/Resources/app/storefront/src
+coverage:
+  status:
+    project:
+      default:
+        threshold: 0.01%
+    patch:
+      default:
+        target: auto
+        threshold: 5%
```

Comment:
I'm not really sure what to pick here. If you take a look at [this example](https://github.com/shopware/shopware/runs/36301091172), it misses the mark by over 20%, 5% would be like one new line not covered

Reply:
lets try